operator/model/pom.xml

```diff
@@ -70,7 +44,6 @@
                 </executions>
                 <configuration>
                     <source>src/main/resources/kubernetes/crd.yml</source>
-                    <extraAnnotations>true</extraAnnotations>
```

Comment:
giving up on trying to keep the Builders and Fluent API, it's too much of a burden to form a sane classpath with `sundrio`.